src/catch2/interfaces/catch_interfaces_test_invoker.hpp

```diff
@@ -12,6 +12,8 @@ namespace Catch {
 
     class ITestInvoker {
     public:
+        virtual void testCaseStarting();
+        virtual void testCaseEnding();
```

Comment:
Same note about name here.

Reply:
I have named these `prepareTestCase` and `tearDownTestCase` now